engine/include/cubos/engine/imgui/serialization.hpp

```diff
@@ -1,12 +1,14 @@
 /// @file
 /// @brief Functions for showing and editing serializable objects in the UI.
-/// @ingroup core-ui
+/// @ingroup imgui-plugin
 
 #pragma once
 
 #include <cubos/core/data/package.hpp>
 
-namespace cubos::core::ui
+using Package = cubos::core::data::Package;
+
+namespace cubos::engine::tools
```

Comment:
```suggestion
namespace cubos::engine
```
imguiPlugin isn't part of the tools namespace.